src/docs/testing/unit-testing.md

```diff
@@ -29,14 +29,34 @@ import { MyCmp } from '../my-cmp';
 it('should render my component', async () => {
   const page = await newSpecPage({
     components: [MyCmp],
-    html: `<my-cmp></my-cmp>`,
+    html: `<my-cmp title="Hello World"></my-cmp>`,
   });
   expect(page.root).toEqualHtml(`
-    <my-cmp>Success!</my-cmp>
+    <my-cmp>Hello World</my-cmp>
   `);
 });
 ```
 
+The example below uses the template option to test the component
+```
```

Comment:
Could we please update this line to denote this is a tsx file? That should help give some rendering engines an idea as to specific keywords it can highlight (similar to the snippet starting on line 25)

See how the top snippet is highlighted, and the bottom one is in all blue in GitHub's dark view:
![Screen Shot 2021-07-15 at 7 04 19 PM](https://user-images.githubusercontent.com/1930213/125868539-4d438687-9de0-458a-a08b-f5f08e4b8028.png)

To do so, I think we just need to add `tsx` to this line like so:
```suggestion
```tsx
```
```